### ISSUE TITLE
Bad build in docker hub for latest image

### ISSUE THREAD

**Issue by angelfs** [PGBI/kong-dashboard#206]
## Issue summary

Problem with the latest kong-dashboard version in docker hub, I think you build the 3.5 version and not 3.6.

## Your environment

* Kong version = 1.0.2
* Kong Dashboard version = latest (normally 3.6.0 but i think it's 3.5.0)
* Node version = 8.15.0
* Npm version = 6.4.1

## Issue description

I tried to get the latest kong-dashboard with the latest kong but I get an error message.
`docker-compose logs kong-dashboard` :
"This version of Kong dashboard doesn't support Kong v0.15 and higher."

After a little search, I found that your build version latest in docker hub correspond to the version 3.5 and not 3.6. 
Just rebuild with the good version ;)

### Current behavior

Kong-dashboard can't connect to my kong

### Expected behavior

If kong is in version 1.0.2 and kong-dashboard in 3.6.0, all it's ok. I have clone your git repo and build with tag 3.6 and all it's ok.

### Steps to reproduce

With docker-compose:
Install latest kong version then install latest kong-dashboard version and you get this error.

## Good to have

Any screenshots? Browsers' logs?

**Comment by jihoon-seo** [comment]
1. My docker images status:
I deleted the docker image `kongdashboard_kong-dashboard`, and kept the docker images `kong`, `postgres`, and `node`.
```shell
# docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
kong                1.0.3               5aaacee41330        4 weeks ago         93.3MB
postgres            9.5-alpine          ef0e648682d3        4 weeks ago         36.9MB
node                10.9-alpine         df2d34f007a1        7 months ago        70.6MB
```

2. My docker containers status:
I stopped and removed all docker containers.
```shell
# docker ps -a
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
```

3. `git pull` from `kong-dashboard` repo, to fetch any newer commits.
```shell
# git pull
Already up to date.
```

```shell
# git status
On branch 3.0
Your branch is up to date with 'origin/3.0'.

nothing to commit, working tree clean
```

```shell
# git log | head
commit 7f67701dc1e06266f92c7cb9291c553de4eae2b6
Author: PGBI <pbarrau@classy.org>
Date:   Sat Mar 16 17:08:33 2019 -0700

    3.6.0

commit b61ff08fba55472cb4db9b553a4a1a4ad7f5d1b8
Merge: e93365a 0b1096f
Author: Pierre <pierre.barrau@mines-paris.org>
Date:   Sat Mar 16 17:06:21 2019 -0700
...
```

4. With `KONG_VERSION=1.0.3 docker-compose up kong-dashboard` command, all is well.
  - Note: `KONG_VERSION=1.0.3-alpine` also works well.
  - Note: By now, KONG_VERSION=1.1.x` does not work.
```shell
# KONG_VERSION=1.0.3 docker-compose up kong-dashboard
...
kong-dashboard    | serving
kong-dashboard    | kong-dashboard stdout: Connecting to Kong on http://kong:8001 ...
kong-dashboard    | kong-dashboard stdout: Connected to Kong on http://kong:8001.
kong-dashboard    | Kong version is 1.0.3
kong-dashboard    | kong-dashboard stdout: Starting Kong Dashboard on port 8080
kong-dashboard    | kong-dashboard stdout: Kong Dashboard has started on port 8080
```

5. `kong-dashboard` web UI says that its version is `v3.6.0`
```
Kong node: http://kong:8001 v1.0.3 — Kong-Dashboard v3.6.0
```

6. In `docker-compose logs kong-dashboard`, I cannot find the error message
"This version of Kong dashboard doesn't support Kong v0.15 and higher."

---

I tested again in a clean environment, and the result is the same.
1. Create a Ubuntu 18.04.2 VirtualBox VM.
2. Install `git` and `docker-compose`.
3. `# git clone https://github.com/PGBI/kong-dashboard.git && cd kong-dashboard`
4. `# KONG_VERSION=1.0.3-alpine docker-compose up kong-dashboard`
5. All is well. (and the version of the `kong-dashboard` pulled from Docker Hub is `v3.6.0`.)

---

FYI: I found that kong-dashboard `3.6.0` does not work with Kong `1.1.x`.

So I tried with Kong `1.0.3` and `1.0.3-alpine`, which works fine with kong-dashboard `3.6.0`.

**Comment by angelfs** [comment]
Yes, you're right, kong dashboard v3.6.0 works only with kong `1.0.*`
However, after some tests, I found that kong-dashboard 3.6.0 doesn't work correctly with kong 1.0.*
Problem with key-auth, services and routes duplication.
If you want use kong `1.1.*`, I suggest you to use konga.
